e2e/tests/kube_client.go

```diff
@@ -0,0 +1,43 @@
+//go:build e2e
```

Comment:
Because the e2e tests depend on a Kubernetes environment to be available, we isolate the e2e tests with a build tag so they're ignored unless the test command is run with `-tags=e2e`. Issue #581 covers adding the E2E tests to our Github Actions.

Reply:
Thoughts on copy-pasting your comment in the README?